ios-target/Networking/Services/TargetAPI.swift

```diff
@@ -21,17 +22,24 @@ class TargetAPI {
     }, failure: failure)
   }
   
-  // TODO:
-//  class func createTarget(_ success: @escaping ([Target]) -> Void, failure: @escaping (_ error: Error) -> Void) {
-//    APIClient.request(.get, url: targetsUrl, success: { response, _ in
-//      do {
-//        guard let targets = response["targets"] as? [Any], !targets.isEmpty else {
-//          failure(App.error(domain: .parsing, localizedDescription: "Could not parse valid targets".localized))
-//          return
-//        }
-//        success(Target.parse(targets: targets))
-//      }
-//    }, failure: failure)
-//  }
-  
+  class func createTarget(_ targetTitle: String, targetArea: Double, targetTopic: Int, targetLocation: CLLocationCoordinate2D, success: @escaping (_ response: Target) -> Void, failure: @escaping (_ error: Error) -> Void) {
```

Comment:
pass a Target here to avoid passing so many parameters

Reply:
It seems a bit counter-intuitive to pass a `Target` to a method that is intended to create the `Target`. Also it would require to add dummy ids in the instances. 
Maybe area and location can be joined as a Region 🤔

Reply:
HA! good point, but the method is intended to save the target in the backend, you already have all the data to create it I don;t think its a bad pattern

Reply:
It is not bad, If we were using offline storage it would be a good approach actually. Just saying that  it seems a bit odd 🙈